src/Service/CloudWatchLogs/tests/Unit/Input/CreateLogStreamRequestTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Tests\Unit\Input;
+
+use AsyncAws\CloudWatchLogs\Input\CreateLogStreamRequest;
+use AsyncAws\Core\Test\TestCase;
+
+class CreateLogStreamRequestTest extends TestCase
+{
+    public function testRequest(): void
+    {
+        $input = new CreateLogStreamRequest([
+            'logGroupName' => 'my-log-group',
+            'logStreamName' => 'my-log-stream',
+        ]);
+
+        // see https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_CreateLogStream.html
+        $expected = '
+POST / HTTP/1.0
+Content-Type: application/x-amz-json-1.1
+X-Amz-Target: Logs_20140328.CreateLogStream
+{
```

Comment:
This should be a valid HTTP response: headers and body are separated by a blank line

```suggestion
X-Amz-Target: Logs_20140328.CreateLogStream

{
```

Reply:
Oh I'm dumb, thanks.